sdk/nodejs/automation/stack.ts

```diff
@@ -885,6 +888,10 @@ export interface PreviewOptions extends GlobalOpts {
     parallel?: number;
     message?: string;
     expectNoChanges?: boolean;
+    /**
+      * Refresh the state of the stack's resources against the cloud provider before running preview.
+      */
+    refresh?: boolean;
```

Comment:
minor: I don't know if this will affect the doc extraction, but the `*` is aligned with the first in other cases:
```suggestion
     * Refresh the state of the stack's resources against the cloud provider before running preview.
     */
    refresh?: boolean;
```

Ignore if this doesn't affect doc extraction.